Show config actions and a result as Console output

christopher-hopper opened the pull request:
## Proposed Changes

1. Output to console config changes being made
1. When `-v` (verbose) flag is used, also output PHP CodeSniffer config command results

## Related Issues

Fixes #7

Potherca commented on src/Plugin.php:
```diff
@@ -140,18 +140,29 @@ private function loadInstalledPaths()
-            $arguments = ['--config-set', 'installed_paths', implode(',', $this->installedPaths)];
+            $paths = implode(',', $this->installedPaths);
+            $arguments = ['--config-set', $configKey, $paths];
+            $this->io->write(sprintf('PHP CodeSniffer Config <info>%s</info> <comment>set to</comment> <info>%s</info>', $configKey, $paths));
```
As a matter of cleanup, I would suggest assigning the message to a separate variable (instead of calling `$this->io->write()`) and placing the call to io->write outside of the if/else.

frenck commented on src/Plugin.php:
```diff
@@ -141,18 +141,29 @@ private function loadInstalledPaths()
     {
-        // By default we delete the installed paths
-        $arguments = ['--config-delete', 'installed_paths'];
+        $configKey = 'installed_paths';
```
This `configKey` (`installed_paths`) is used multiple times in this class / Plugin (e.g. line 121).

I would really like to keep a single style. Right now it used as a variable here, while being used directly elsewhere. I would suggest on moving it into a class constant.

frenck commented on src/Plugin.php:
```diff
@@ -141,18 +141,29 @@ private function loadInstalledPaths()
+            $arguments = ['--config-set', $configKey, $paths];
+            $this->io->write(sprintf('PHP CodeSniffer Config <info>%s</info> <comment>set to</comment> <info>%s</info>', $configKey, $paths));
+        }
+        else {
```
Violates [PRS-2: 5.1. if, elseif, else](http://www.php-fig.org/psr/psr-2/#if-elseif-else).
Please correct and place `else` on the same line as the closing bracket of the above if statement.

christopher-hopper commented:
Thanks for the review @frenck @Potherca -- I've pushed updates addressing your suggestions. Love your work.

I was a little unsure about the constant name. Do I go for the better readability, or more brevity. Landed somewhere in the middle I hope. *Naming things*: so difficult sometimes.

Potherca commented:
@christopher-hopper To (mis)quote [a poem by T.S. Eliot][1]:

>  The Naming of <del>Cats</del> <ins>a constant</ins> is a difficult matter,
> It isn't just one of your holiday games;
> You may think at first I'm as mad as a hatter
> When I tell you, a <del>cat</del> <ins>a constant</ins> must have THREE DIFFERENT NAMES.

😁

[1]: https://en.wikipedia.org/wiki/The_Naming_of_Cats
